uaclient/entitlements/base.py

```diff
@@ -28,6 +28,10 @@
 
 
 class UAEntitlement(metaclass=abc.ABCMeta):
+
+    # Optional URL for top-level product service information
+    help_doc_url = ""
```

Comment:
Why `""` and not `None`?

Reply:
^ Lazy developer. Fixed to None and declared # type: str. I was working around mypy errors claiming invalid type str on subclasses that properly set this attribute. Turns out I only needed to declare the type on the base class.